nonstandard/QA_scripts/change_check_cb.R

```diff
@@ -1,14 +1,16 @@
 #-------------- Change checking helper
```

Comment:
ASAP I'd love to integrate these tests into the testthat framework. I won't spend time now trying to understand how it works but whenever we have that time I'd love to do it and help.

Reply:
totally, my vision is abstracting calculations from IO and then implement snapshot tests on a wide range of configurations, happy to tackle this soon.